content/blog/2018/egress-monitoring-access-control.md

```diff
@@ -0,0 +1,546 @@
+
+The [Collecting Metrics and Logs](/docs/tasks/telemetry/metrics-logs/) task describes how to configure metrics and logs for services in a mesh. The [Visualizing Metrics with Grafana](/docs/tasks/telemetry/using-istio-dashboard/) describes the Istio Dashboard to monitor mesh traffic. The [Basic Access Control](/docs/tasks/security/basic-access-control/) task shows how to control access to in-mesh services. The [Secure Access Control](http://localhost:1313/docs/tasks/security/secure-access-control/) task shows how to configure access policies using black or white list checkers. As opposed to the telemetry and security tasks above, this blog post describes Istio's monitoring and access policies applied exclusively to the egress traffic.
+
+## Before you begin
+
+Follow the steps in the [Configure an Egress Gateway, Perform TLS origination with the egress Gateway](/docs/tasks/traffic-management/egress-gateway/#perform-tls-origination-with-the-egress-gateway) task, without the [Cleanup](/docs/tasks/traffic-management/egress-gateway/#cleanup) step. After you accomplish this, you will be able to access [edition.cnn.com/politics](https://edition.cnn.com/politics) from an in-mesh container that has _curl_ installed. In the instructions of this blog post we assume that the `SOURCE_POD` environment variable contains the pod name.
+
+## Configure monitoring and access policies
+
+Note that since we want to accomplish our tasks in a _secure way_, we must direct egress traffic through _egress gateway_, as described in the [Configure an Egress Gateway](/docs/tasks/traffic-management/egress-gateway/) task. The _secure way_ here means that we want to prevent malicious applications from bypassing Istio monitoring and policy enforcement.
+
+In our scenario, the organization performed the instructions in the [Before you begin](#before-you-begin) section. It enabled traffic to _edition.cnn.com_ and configured that traffic to pass through the egress gateway. Now it is ready to configure Istio for monitoring and access policies for the traffic to _edition.cnn.com_.
+
+### Logging
+
+Let's configure Istio to log access to _*.cnn.com_. We create a `logentry` and two [stdio](/docs/reference/config/policy-and-telemetry/adapters/stdio/) `handlers`, one for logging forbidden access (_error_ log level) and another one for logging all access to _*.cnn.com_ (_info_ log level). Then we create `rules` to direct our `logentry` instances to our `handlers`. One rule directs access to _*.cnn.com/politics_ to the handler for logging forbidden access, another rule directs log entries to the handler that outputs each access to _*.cnn.com_ as an _info_ log entry. To understand the Istio `logentries`, `rules`, and `handlers`, see [Istio Adapter Model](/blog/2017/adapter-model/). A diagram with the involved entities and dependencies between them appears below:
```

Comment:
Throughout the doc, please use "You" instead of "We". 

https://istio.io/about/contribute/style-guide/#address-the-reader-as-you